docs/calico.md

```diff
@@ -80,6 +80,12 @@ you'll need to edit the inventory and add a hostvar `local_as` by node.
 node1 ansible_ssh_host=95.54.0.12 local_as=xxxxxx
 ```
 
+In another cases you may want to utilize `peer_with_router` with NAT.
+Then the following variables need to be set:
+
+* `nat_outgoing` to enable NAT (default value: true).
+* `peer_with_router_nat_outgoing` to forcefully enable NAT (default value: false).
```

Comment:
```
to forcefully enable NAT (default value: false).
```
seems to make confusions because `nat_outgoing` is to enable NAT and the default value is `true`.
That means the NAT is already enabled by default.